File: Studio.API/Business/Domain/Entities/Weddings/Wedding.cs
using Studio.API.Business.Domain.Entities.Bases;
using Studio.API.Business.Domain.Entities.Events;
using System.ComponentModel.DataAnnotations.Schema;

namespace Studio.API.Business.Domain.Entities.Weddings
{
    [Table("Wedding")]
    public class Wedding : BaseEntity
    {
        [Column(TypeName = "nvarchar(255)")]
        public string WeddingTittle { get; set; }
        
        [Column(TypeName = "nvarchar(255)")]
        public string? WeddingDescription { get; set; }
        
        public string? Status { get; set; }
        
        public DateTime StartDate { get; set; }
        
        public DateTime EndDate { get; set; }   

        public virtual IList<Event> Events { get; set; }
    }
}

File: Studio/src/app/data/entity/Wedding.ts
import { BaseEntity } from "./BaseEntity";
import { EventEntity } from "./Event";

export interface Wedding extends BaseEntity{
    weddingTittle: string;
    weddingDescription?: string | null;
    status?: string | null;
    startDate: Date;
    endDate: Date;
    events: EventEntity[];
}